src/components/CategoryPopover/CategoryPopoverContent/CategoryPopoverContent.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react'
+import * as Styled from './styles'
+// import Button from '../Button'
+
+const CategogyPopoverContent = () => {
```

Comment:
It's not necessary to place this logic into a separate `CategogyPopoverContent` component (but it's ok). It can be placed in `CategoryPopover` component.